eth/chains/ropsten/__init__.py

```diff
@@ -34,8 +34,8 @@
 
 
 class BaseRopstenChain:
-    vm_configuration = ROPSTEN_VM_CONFIGURATION  # type: Tuple[Tuple[int, Type[BaseVM]], ...]  # noqa: E501
-    chain_id = ROPSTEN_CHAIN_ID  # type: int
+    vm_configuration: Tuple[Tuple[int, Type[BaseVM]], ...] = ROPSTEN_VM_CONFIGURATION  # noqa: E501
```

Comment:
I bet the line is short enough now that we can drop the `noqa`

Reply:
Right